apps/ledger-live-mobile/src/families/cardano/Delegations/index.tsx

```diff
@@ -177,9 +179,9 @@ function Delegations({ account }: Props) {
                 {
                   label: t("cardano.delegation.drawer.rewards"),
                   Component: (
-                    <LText numberOfLines={1} semiBold style={[styles.valueText]}>
-                      {delegation.rewards.toString() ?? ""}
-                    </LText>
+                    <Text variant={"body"} fontWeight={"semiBold"}>
+                      <CurrencyUnitValue value={delegation.rewards} unit={unit} />
+                    </Text>
```

Comment:
To be checked but github bot added this tooltip : 

`React Hook useMemo has a missing dependency: 'unit'. Either include it or remove the dependency array.`

Do you think we need to add unit in the `[delegation, t, account, onOpenExplorer] `array ?

Reply:
cc @mehulcs